src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CollectionExpression.cs

```diff
@@ -138,6 +141,22 @@ static BoundNode unwrapListElement(BoundCollectionExpression node, BoundNode ele
             }
         }
 
+        private bool IsSingleReadOnlySpanSpread(BoundCollectionExpression node, [NotNullWhen(true)] out BoundExpression? spreadExpression)
+        {
+            spreadExpression = null;
+
+            if (node.Elements is
+                [
+                    BoundCollectionExpressionSpreadElement { Expression: { Type: NamedTypeSymbol spreadType } expr, IteratorBody: not BoundExpressionStatement { Expression: BoundConversion { ConversionKind: not ConversionKind.Identity } } }
```

Comment:
I think we might really do wonders for readability if, at the least, we broke out `Expression:` and `IteratorBody:` so that they are on the start of new lines.

Reply:
> we broke out `Expression:` and `IteratorBody:` so that they are on the start of new lines.

I've already tried this way before your comment, but unfirtunatelly formatter doesn't like this :(
![devenv_LrcHWS9lAw](https://github.com/dotnet/roslyn/assets/70431552/5d917f86-8030-4fa1-8ed5-530768e96e00)

Reply:
Let's check for an existing bug about this on the formatter or file a new bug report. And in the mean time I think disabling format around this check with pragma would be reasonable.

Reply:
Also, we unfortunately haven't clearly indicated this guideline, but please refrain from hitting "resolve" on comment threads from reviewers, but allow the reviewer to hit resolve, in order to let the reviewer weigh in on whether the feedback is resolved, and also make it easier to see the actual reply which explains the resolution. Feel free to use +1 reaction or a reply to indicate that you believe you have resolved the feedback.

Reply:
> Let's check for an existing bug about this on the formatter or file a new bug report

Filled https://github.com/dotnet/roslyn/issues/73251